src/commands/NewIntegrationFileCommand.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+import * as fs from 'fs';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import * as kamel from './../kamel';
```

Comment:
`./` seems obsolete. `../kamel` should be actually enough?

Reply:
It's in a deeper folder, so the path has to climb an additional level. I think it's ok

Reply:
Brian is right. It is required as it is in a deeper folder

Reply:
`./` is the current folder
`./../` is a complicated way to write `../`

Reply:
my bad, I read too fast the feedback.
Will need to look if there is adifference or a recommended way in TypeScript but effectively `../` is easier to read

Reply:
It is really no black magic. Its just the way it works on any operating system. Go to a shell and type both and you will see u end up in the same folder.

Reply:
it is not that straightforward. In examples on Typescript language, they are using `./` notation for same level files https://www.typescriptlang.org/docs/handbook/modules.html
I can understand that to stay coherent with that, a good practice would be to keep it.

I don't have opinion on that. Would need to find reference.

Reply:
Yeah, I see your point. Now I am interested too.

Reply:
Check out https://www.typescriptlang.org/docs/handbook/module-resolution.html

Reply:
![image](https://user-images.githubusercontent.com/435853/78711669-a5238a80-7917-11ea-8a09-7d56657d9064.png)

Reply:
cool, that's a good reference